Components/IconViews/Bar/Config.xml

```diff
@@ -140,6 +140,56 @@
 					</OnLoad>
 				</Scripts>
 			</Slider>
+
+			<Slider parentKey="SizeX" inherits="TellMeWhen_SliderTemplate">
+				<Anchors>
+					<Anchor point="TOP" relativeKey="$parent.BorderIcon" relativePoint="BOTTOM"  y="-20"/>
+					<Anchor point="LEFT" relativeKey="$parent.BorderIcon" />
+					<Anchor point="RIGHT" relativeKey="$parent.BorderIcon" />
+					
+				</Anchors>
+				<Scripts>
+					<OnLoad>						
+						TMW:CInit(self)
+
+						self:SetTexts(TMW.L["UIPANEL_BAR_SIZE_X"], TMW.L["UIPANEL_BAR_SIZE_X_DESC"])
+						self:SetSetting("SizeX")
+
+						self:SetMode(self.MODE_ADJUSTING)
+						self:SetMinMaxValues(-2000, 2000)
```

Comment:
Sorry, another one - negative size is nonsensical. I would say the absolute minimum here should be `1` (and for Y as well). I promise this is the last thing.